plugins/rapid7_insightidr/komand_rapid7_insightidr/actions/query/action.py

```diff
@@ -19,16 +20,22 @@ def __init__(self):
         )
 
     def run(self, params={}):
+        most_recent_first = params.get(Input.MOST_RECENT_FIRST)
         time_now = int(time.time())
         request = ResourceHelper(self.connection.session, self.logger)
         # 7776000 - is for three months from now.
         # It is here because InsightDR keep logs for three months in hot storage
         three_months_seconds = 7776000
-        request_params = {"from": (time_now - three_months_seconds) * 1000, "to": time_now * 1000}
+        # The way data indexing works changed on the 24/11/2022.
+        # For any search with most_recent_first=true 'from' must not be older than 24/11/2022
+        twenty_fourth_november = 1669248000
+        if (time_now - three_months_seconds) > twenty_fourth_november:
+            from_var = time_now - three_months_seconds
+        else:
+            from_var = twenty_fourth_november
```

Comment:
What in the case we set `most_recent_first` to `False`? It will still count for less than 3 months 😄